components/modal/index.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+
+export default function Modal({ open, close, children }) {
+  if (!open) return null;
```

Comment:
I think it would be better, to just having this state outside of the `Modal` component and just mount the component, when `open` is true.